types/heremaps/index.d.ts

```diff
@@ -5694,7 +5694,7 @@ declare namespace H {
              * @param opt_locale {(H.ui.i18n.Localization | string)=} - the language to use (or a full localization object).
              * @returns {H.ui.UI} - the UI instance configured with the default controls
              */
-            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes | H.service.MapType | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
+            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
```

Comment:
I took a look at the API and it looks like it actually _does_ accept a `DefaultLayers`. For the most part it seems to only explicitly look for the `normal`, `satellite`, and `terrain` map types, but also uses the `incidents` layer as well if present. It seems that the API is a bit looser than the documentation specifies.